packages/libs/src/sdk/sdk.ts

```diff
@@ -87,9 +92,16 @@ const initializeServices = (config: SdkConfig) => {
     )
   }
 
-  const defaultAuthService = config.apiKey
-    ? new AppAuth(config.apiKey, config.apiSecret)
-    : new Auth()
+  let auth
+  if (config.services?.auth) {
+    auth = config.services?.auth
+  } else if (config.apiKey && config.apiSecret) {
+    auth = new AppAuth(config.apiKey, config.apiSecret)
+  } else if (config.userAuth) {
+    auth = new HedgehogAuth(defaultHedgehogAuthConfig)
+  } else {
+    auth = new DefaultAuth(config.apiKey)
+  }
```

Comment:
nit: could use an expression + const here to prevent further mutation of `auth`
```suggestion
const getAuth = () => {
  if (config.services?.auth) {
    return config.services?.auth
  } else if (config.apiKey && config.apiSecret) {
    return new AppAuth(config.apiKey, config.apiSecret)
  } else if (config.userAuth) {
   return new HedgehogAuth(defaultHedgehogAuthConfig)
  } else {
    return new DefaultAuth(config.apiKey)
  }
}
const auth = getAuth()

```

Reply:
<3 <3